binderhub/app.py

```diff
@@ -207,6 +207,14 @@ def _valid_badge_base_url(self, proposal):
         config=True,
     )
 
+    registry_class = Type(
```

Comment:
What do you think of renaming it to `docker_registry_class`? Just to be a bit more explicit.

Reply:
Sure thing. I will refactor and re-test as soon as possible.

Reply:
Done!